CMakeLists.txt

```diff
@@ -2646,7 +2646,7 @@ if(APPLE OR WIN32)
   # qt_de.qm is just one arbitrary file in the directory that needs to be located;
   # there is no particular reason to look for this file versus any other one in the directory.
   if(QT6)
-    find_file(QT_TRANSLATION_FILE qt_de.qm PATHS "${Qt6_DIR}/../../../translations" "${Qt6_DIR}/../../qt5/translations" REQUIRED NO_DEFAULT_PATH)
```

Comment:
Side note: `QT_DIR` seems to work too, as well as simply passing `/translations/Qt6` (which seems to be implicity resolved against `CMAKE_PREFIX_PATH`, i.e. the vcpkg install root, even if `NO_DEFAULT_PATH` is passed?!).

For consistency I went with `Qt6_DIR` though.